tests/globalhelper/pod.go

```diff
@@ -57,3 +59,41 @@ func GetListOfPodsInNamespace(namespace string) (*corev1.PodList, error) {
 
 	return runningPods, nil
 }
+
+// CreateAndWaitUntilPodIsReady creates a pod and waits until all its containers are ready.
+func CreateAndWaitUntilPodIsReady(pod *corev1.Pod, timeout time.Duration) error {
```

Comment:
Please move CreateAndWaitUntilPodIsReady from cnfcert-tests-verification/tests/lifecycle/lifehelper/lifehelper.go to the globalhelper, reuse and refactor lifecycle code

Reply:
Do you mind if I do this refactor in a new PR, please? That refactor would affect other test suites, creating a lot of changes not directly related to this PR, thus making further readings/reviews of this PR/git commit harder.

Reply:
Please move these function in this PR. There is a risk that we forget about refactoring and add code duplication in the project.